pkgs/tools/networking/trurl/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, runCommand, curl, perl }:
```

Comment:
Is `runCommand` required here? I tried it without and I'm able to build successfully.

Reply:
Yeah, it is not required, it's merely a leftover.
Thanks, fixed!